templates/compositions/ec-component-icons-with-text/icons-with-text.html.twig

```diff
@@ -0,0 +1,46 @@
+{% spaceless %}
+
+{#
+  /**
+   * @file
+   * Theme implementation for a ECL list of icons with text.
+   */
+Parameters:
+- "items" (array) (default: [])
+  - "icon" (string) (default: '')
+  - "text" (string) (default: '')
+#}
+
+{# Internal properties #}
+
+{% set _icon_path = icon_path|default('') %}
+{% set _extra_classes = 'ecl-u-mr-s' %}
+
+{# Internal logic - Process properties #}
+
+{% set _items = [] %}
+{% for item in  items %}
+  {% if item.icon and item.text %}
+    {% set _items = _items|merge([item]) %}
+  {% endif %}
+{% endfor %}
```

Comment:
What does this do? Can't we simply loop trough `items` down below?

Reply:
Removed.